packages/playwright-core/src/server/registry/download.ts

```diff
@@ -99,14 +100,10 @@ export async function download(
     }
     const errorMessage = error?.message || '';
     log(`attempt #${attempt} - ERROR: ${errorMessage}`);
-    if (
-      attempt < retryCount &&
-      (errorMessage.includes('ECONNRESET') ||
-        errorMessage.includes('ETIMEDOUT'))
-    ) {
+    if (attempt < retryCount) {
       // Maximum default delay is 3rd retry: 1337.5ms
       const millis = Math.random() * 200 + 250 * Math.pow(1.5, attempt);
-      log(`sleeping ${millis}ms before retry...`);
+      log(`sleeping ${millis.toFixed(2)}ms before retry...`);
```

Comment:
Sleeping was introduced as a heuristic: we noticed that our primary CDN was sometimes returning ETIMEDOUT or ECONNRESET for just-uploaded browser builds. So the idea was to wait a little in this case and then retry.

Since you have a bunch of different URLs now, sleeping does not make much sense to me and probably can be nuked